src/.nuget/Microsoft.NETCore.Crossgen/runtime.Linux.Microsoft.NETCore.Crossgen.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CpaotCrossgenRuntimeConfigFile>$(ArtifactsObjDir)crossgen2\crossgen.runtimeconfig.json</CpaotCrossgenRuntimeConfigFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\crossgen" />
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\crossgen.dll" />
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\ILCompiler*.dll" />
```

Comment:
What is the `*` expected to pick up? (If the list is small enough, it may be a good idea to specify them so the build fails if they don't exist, e.g. due to some silent error when building them.)

Reply:
Good point. They're only 3 files currently, but what I was worried about is some refactoring that happens producing a new ILCompiler* dll.

Reply:
Up to you, I agree that could get annoying. 🙂

Reply:
Ok, let's just leave it the way it is then :)